helpers/deserialize.js

```diff
@@ -0,0 +1,31 @@
+module.exports = deserialize
```

Comment:
can you rename to `deserialise`, because we use British English, at least we try as good as we can :)